src/istgt_md5.c

```diff
@@ -61,7 +61,7 @@ istgt_md5final(void *md5, ISTGT_MD5CTX *md5ctx)
 }
 
 int
-istgt_md5update(ISTGT_MD5CTX *md5ctx, const void *data, size_t len)
+istgt_md5update(ISTGT_MD5CTX *md5ctx, const void *data, int len)
```

Comment:
If you are planning to change `size_t` to `int` then change it to `ssize_t` as `size_t` is `unsigned long int` and `ssize_t` is `signed long int`. Other way is to change the condition `len <= 0` to` len == 0`.

Reply:
I didn't change the condition because I didn't know if negative `len` can actually occur or not. This is why I tried to change the data type.

Reply:
can you please change size_t to ssize_t rather than int?

Reply:
Okay. Will do.

Reply:
Done.